app/services/catalog/determine_task_relevancy.rb

```diff
@@ -5,18 +5,16 @@ def initialize(topic)
     end
 
     def process
-      Insights::API::Common::Request.with_request(order_item_context) do
```

Comment:
We should be able to get rid of the `order_item_context` method if we're removing this, right?

Should we be getting rid of where we set the `order_item.context` ([here](https://github.com/RedHatInsights/catalog-api/blob/master/app/models/order_item.rb#L27))? I couldn't find anywhere else that uses it, aside from the `add_to_order_spec.rb` which implicitly goes through the `set_defaults` method when an order_item gets created, but if we don't care about accessing it any time, maybe we don't care about setting it either?